applications/serial_lte_modem/doc/FOTA_AT_commands.rst

```diff
@@ -47,10 +49,17 @@ Syntax
 
 .. note::
 
-   When doing modem FOTA, erasing the modem DFU area is optional since the update process will automatically erase the area if needed.
+   When doing modem delta FOTA, erasing the modem DFU area is optional since the update process will automatically erase the area if needed.
 
    However, this leads to the command starting the update taking longer to complete, and also leaves the connection to the FOTA server idle while the area is being erased, which can provoke issues.
 
+.. note::
+
+   When doing full modem FOTA, external flash is used to store the firmware.
```

Comment:
```suggestion
   The firmware image is stored in external flash memory during a full modem update over FOTA.
```

Reply:
Fixed, but the FOTA appreviation has been used.

Reply:
Perhaps 'using FOTA', then.

Reply:
Fixed accordin to the teams chat